applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h

```diff
@@ -538,23 +497,7 @@ namespace Kratos {
                                             TSystemVectorType& Dx,
                                             TSystemVectorType& b) override
         {
-            ProcessInfo& CurrentProcessInfo = r_model_part.GetProcessInfo();
-
-            Scheme<TSparseSpace, TDenseSpace>::InitializeSolutionStep(r_model_part, A, Dx, b);
-
-            double DeltaTime = CurrentProcessInfo[DELTA_TIME];
-
-            if (DeltaTime == 0)
-                KRATOS_THROW_ERROR(std::logic_error, "detected delta_time = 0 in the Bossak Scheme ... check if the time step is created correctly for the current model part", "");
-
-            //initializing constants
-            ma0 = 1.0 / (mGammaNewmark * DeltaTime);
-            ma1 = DeltaTime * mBetaNewmark / mGammaNewmark;
-            ma2 = (-1 + mGammaNewmark) / mGammaNewmark;
-            ma3 = DeltaTime;
-            ma4 = pow(DeltaTime, 2)*(-2.0 * mBetaNewmark + 1.0) / 2.0;
-            ma5 = pow(DeltaTime, 2) * mBetaNewmark;
-            mam = (1.0 - mAlphaBossak) / (mGammaNewmark * DeltaTime);
+            BaseType::InitializeSolutionStep(r_model_part, A, Dx, b);
```

Comment:
better use the `using` syntax if you only call the baseclass

Reply:
I don't know the advantage of using "using" syntax over typedef in here. :/ Is there an advantage? @philbucher

Reply:
This function does nothing except calling the baseclass => hence it should be removed
try if it compiles, if not then I can explain you in person what I meant

Reply:
Ah, Now I see... It was an empty method calling the base class, which I forgot to remove. I removed it now :) Thanks